engine/src/physics/solver/penetration_constraint/plugin.cpp

```diff
@@ -266,10 +290,12 @@ void cubos::engine::penetrationConstraintPlugin(Cubos& cubos)
                 float frictionMass = kFriction > 0.0F ? 1.0F / kFriction : 0.0F;
 
                 // determine friction (set to predefined value for now)
```

Comment:
`(set to predefined value for now)`

Is this part of the comment still supposed to be here, now that friction/restitution are customizable in the material?

Reply:
No it isn't, nice catch